kitchen-tests/Gemfile.lock

```diff
@@ -59,9 +59,8 @@ GEM
     excon (0.51.0)
     faraday (0.9.2)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.14)
```

Comment:
Looking at the whole gemfile.lock I think thats the right thing. windows is the only platform that needs that gem and it kept the mingw version